path_config

```diff
@@ -16,6 +16,11 @@ path /status StatusHandler {
     root .;
 }
 
+path /reverse_proxy ReverseProxyHandler {
+    remote_host localhost;
+    remote_port 1234;
```

Comment:
This seems correct per the assignment spec, but after the discussion on friday with our TA, it seems like our reverse proxy is going to be part of our web server, instead of a separate instance. That is, it will always be listening on the same port as the rest of the server. So I don't think we would/should need to configure a port inside reverse proxy's child block.